docs/content/topics/project-developers/secrets.md

```diff
@@ -9,239 +9,157 @@ aliases:
   - /topics/project-developers/secrets.md
 ---
 
-TODO: update per v2
-
 # Secret Management
 
-Brigade provides tools for storing sensitive data outside of your `brigade.js` scripts,
-and then passing that information into the jobs that need them. Brigade accomplishes
-this by making use of Kubernetes secrets.
+Brigade provides tools for storing sensitive data outside of your Brigade
+scripts, and then passing that information into the jobs that need them.
+
+This is accomplished by associating secrets with a given project, which the
+project's script can then access as needed. Secrets are persisted on the
```

Comment:
Can we emphasize that secrets live _only_ on the substrate and are never stored in the database?